.github/workflows/govulncheck.yaml

```
@@ -0,0 +1,26 @@
+name: govulncheck
+
+on:
+  push:
+    branches:
+      - master
+      - branch/*
+    paths-ignore:
+      - 'docs/**'
+      - 'rfd/**'
+      - '**.md'
```

potentially switch from blocking docs to only allowing go changes?
```suggestion
    paths:
      - '**.go'
      - '**/go.mod'
      - '**/go.sum'
```

This has the disadvantage that we won't get runs on PRs without go changes.  If we want to enforce this via branch protection rules at some point, we'll want it to run on everything and be a no-op when there are no go changes.